vscode_extension/src/providers/AacDefinitionEditorProvider.ts

```diff
@@ -159,17 +161,28 @@ export class AacDefinitionEditorProvider implements vscode.CustomEditorProvider<
      * On messages from the editor, perform the appropriate action.
      */
     private onMessage(panel: vscode.WebviewPanel, document: AacDefinitionsDocument, message: any) {
+        const params = new URLSearchParams(document.uri.query);
+        const isNewDefinition = params.get(queryKeys.QUERY_KEY_NEW) === `${true}`;
```

Comment:
See previous comment:

```suggestion
        const isNewDefinition = params.get(queryKeys.QUERY_KEY_NEW) === "true";
```